### ISSUE TITLE
[FEATURE] Fristenanzeige in Akte: Auswahl offen/erledigt/gelöscht

### ISSUE THREAD

**Issue by stm9x9** [jlawyerorg/j-lawyer-org#1523]
![image](https://user-images.githubusercontent.com/29924198/148272143-6d5f4ec1-9da5-48ec-9452-63faf9cc5b98.png)

Im Laufe eines Aktenlebens kommen jede Menge Fristen zusammen. Um die Übersicht zu verbessern, zb wenn noch weit unten eine offene Frist steht, die nicht ins Fenster paßt,, wäre ein Filter hilfreich. GGf. auch mit Nennung der Zahl der betroffenen Fristen.
Damit könnte man sich gezielt die gerade interessanten Fristen anzeigen lassen.

Werden gelöschte Fristen im System eigentlich wirklich gelöscht oder nur als gelöscht markiert? Wenn letzteres, dann könnte man diese auch damit anzeigen/filtern lassen. Optisch dann durchgestrichen.

**Comment by Dickmaulruessler** [comment]
Ja, offene Sachen sollten - unabhängig vom Datum - immer vor erledigten Sachen stehen.

**Comment by GuidoBo** [comment]
Großartig. Ich finde aber auch eine Sortierung toll: offen (nach zu fern sortiert), dann erledigt (ebenfalls nah zu fern sortiert), dann gelöscht

**Comment by j-dimension** [comment]
> Werden gelöschte Fristen im System eigentlich wirklich gelöscht oder nur als gelöscht markiert?

Werden richtig gelöscht.
